### PR TITLE
fix styling and responsive

### LINKED ISSUE #88: Fix Features Responsive

### PR DESCRIPTION
**Pull request by khaldoon9kh**
I fixed some styling bugs in the Features component.

Before:
image

After:
image

Before:
image

After:
image

### CODE REVIEW

File: src/components/SingleFeature/index.jsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import "./index.css";
 
 const SingleFeature = (props) => {
   return (
-    <Row className="imgclass">
+    <Row className="featContainer justify-content-center align-items-center">
       <Col lg={{ order: props.order }}>
         <img
           className="featuresImg"
```
Comment (fadime-ozdemir) [inline src/components/SingleFeature/index.jsx:10]: you can add fluid class to the img class to make it responsive

**Review by fadime-ozdemir — APPROVED** [review]
you can add fluid class to the img class to make it responsive, instead of .featuresImg {
  max-width: 100%;
  height: 100%;
}

**Review by halitbatur — APPROVED** [review]

**Review by halitbatur — APPROVED** [review]

**Comment by halitbatur** [comment]
@khaldoon9kh please update this with the dev branch and its ready to be merged

**Comment by khaldoon9kh** [comment]
PTAL

**Pull request merged**